recipes/stx/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class STXConan(ConanFile):
+    name = 'stx'
+    homepage = 'https://github.com/lamarrr/STX'
+    license = 'MIT'
+    url = 'https://github.com/conan-io/conan-center-index'
+    description = 'C++17 & C++ 20 error-handling and utility extensions.'
+    generators = 'cmake', 'cmake_find_package'
+    topics = 'error-handling', 'result', 'option', 'backtrace', 'panic'
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    options = {
+        'backtrace': [True, False],
+        'panic_backtrace': [True, False],
+        'override_panic_handler': [True, False],
+        'shared': [True, False],
+        'fPIC': [True, False],
+        'visible_panic_hook': [True, False],
+    }
+    default_options = {
```

Comment:
Looks like an option is missing :thinking:

Reply:
I tried to mirror the same default behavior as the library's CMakeLists.txt,
where the default for one option depends on the value of two others.